ambry-utils/src/test/java/com.github.ambry.utils/TestUtils.java

```diff
@@ -33,6 +33,17 @@
   public static final Random RANDOM = new Random();
 
   /**
+   * Create an array of the given size and fill it with random bytes.
+   * @param size the size of the array to create.
+   * @return the created array.
+   */
+  public static byte[] randomArray(int size) {
```

Comment:
isn't this the same as `getRandomBytes()` in this class?

Reply:
Ah, I seem to have missed that. Removed.